modules/python/python-logmsg.c

```diff
@@ -53,14 +53,21 @@ _get_value(PyLogMessage *self, const gchar *name, gboolean cast_to_bytes)
   const gchar *value = log_msg_get_value_with_type(self->msg, handle, &value_len, &type);
 
   if (!value)
-    return NULL;
+    Py_RETURN_NONE;
```

Comment:
Ok from this patch on, I guess get_value will not return NULL without an exception being set.

Reply:
Yes, I had to fix that as well.